packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/review/load-testing.api.md

```diff
@@ -439,75 +191,35 @@ export interface LoadTestAdministrationClientOptionalParams extends ClientOption
-    getAppComponents(testRunId: string, options?: LoadTestRunClientGetAppComponentsOptionalParams): Promise<LoadTestRunClientTestRunAppComponents>;
-    getServerMetricsConfig(testRunId: string, options?: LoadTestRunClientGetServerMetricsConfigOptionalParams): Promise<LoadTestRunClientTestRunServerMetricConfig>;
-    getTestRun(testRunId: string, options?: GetTestRunOptionalParams): Promise<LoadTestRunClientTestRun>;
-    getTestRunFile(testRunId: string, fileName: string, options?: GetTestRunFileOptionalParams): Promise<LoadTestRunClientTestRunFileInfo>;
-    listMetricDefinitions(testRunId: string, metricNamespace: string, options?: ListMetricDefinitionsOptionalParams): Promise<LoadTestRunClientMetricDefinitionCollection>;
-    listMetricDimensionValues(testRunId: string, name: string, metricname: string, metricNamespace: string, timespan: string, options?: ListMetricDimensionValuesOptionalParams): Promise<LoadTestRunClientDimensionValueList>;
-    listMetricNamespaces(testRunId: string, options?: ListMetricNamespacesOptionalParams): Promise<LoadTestRunClientMetricNamespaceCollection>;
-    listMetrics(testRunId: string, metricname: string, metricNamespace: string, timespan: string, body?: LoadTestRunClientMetricRequestPayload, options?: ListMetricsOptionalParams): PagedAsyncIterableIterator<LoadTestRunClientTimeSeriesElement>;
-    listTestRuns(options?: ListTestRunsOptionalParams): PagedAsyncIterableIterator<LoadTestRunClientTestRun>;
+    getAppComponents(testRunId: string, options?: GetAppComponentsOptionalParams): Promise<TestRunAppComponents>;
+    getServerMetricsConfig(testRunId: string, options?: GetServerMetricsConfigOptionalParams): Promise<TestRunServerMetricConfig>;
+    getTestRun(testRunId: string, options?: GetTestRunOptionalParams): Promise<TestRun>;
+    // Warning: (ae-forgotten-export) The symbol "TestRunFileInfo" needs to be exported by the entry point index.d.ts
+    getTestRunFile(testRunId: string, fileName: string, options?: GetTestRunFileOptionalParams): Promise<TestRunFileInfo>;
+    // Warning: (ae-forgotten-export) The symbol "MetricDefinitionCollection" needs to be exported by the entry point index.d.ts
+    listMetricDefinitions(testRunId: string, metricNamespace: string, options?: ListMetricDefinitionsOptionalParams): Promise<MetricDefinitionCollection>;
+    // Warning: (ae-forgotten-export) The symbol "DimensionValueList" needs to be exported by the entry point index.d.ts
+    listMetricDimensionValues(testRunId: string, name: string, metricname: string, metricNamespace: string, timespan: string, options?: ListMetricDimensionValuesOptionalParams): Promise<DimensionValueList>;
+    // Warning: (ae-forgotten-export) The symbol "MetricNamespaceCollection" needs to be exported by the entry point index.d.ts
+    listMetricNamespaces(testRunId: string, options?: ListMetricNamespacesOptionalParams): Promise<MetricNamespaceCollection>;
```

Comment:
this should be expected, as we will just generate one models for all subclients.